File: database/src/main/java/bao/webapp/model/Order.java
package bao.webapp.model;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;


@Entity(name = "Orders")
public class Order {
	
	@Id
	@GeneratedValue
	private long OrderID;
	
	@Column(name = "Status")
	private String status;
	
	@Column(name = "CreateDate")
	private Date createDate;
	
	@Column(name = "Destination")
	private String destination;
	
	@Column(name = "UserID")
	private long userID;
	
	@Column(name = "BusinessID")
	private long businessID;
	
	@Column(name = "Goods")
	private String goods;
	
	public long getOrderID() {
		return OrderID;
	}
	
	public void setOrderID(Long orderID) {
		this.OrderID = orderID;
	}
	
	public String getStatus() {
		return status;
	}
	
	public void setStatus(String status) {
		this.status = status;
	}
	
	public Date getCreateDate() {
		return createDate;
	}
	
	public void setCreateDate(Date createDate) {
		this.createDate = createDate;
	}
	
	public String getDestination() {
		return destination;
	}
	
	public void setDestination(String destination) {
		this.destination = destination;
	}
	
	public long getUserID() {
		return userID;
	}
	
	public void setUserID(Long userID) {
		this.userID = userID;
	}
	
	public long getBusinessID() {
		return businessID;
	}
	
	public void setBusinessID(Long businessID) {
		this.businessID = businessID;
	}
	
	public String getGoods() {
		return goods;
	}
	
	public void setGoods(String goods) {
		this.goods = goods;
	}
}

File: database/src/main/kotlin/com/ossp/database/model/Order.kt
package com.ossp.database.model

import org.hibernate.annotations.GenericGenerator
import java.util.*
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity(name = "Order")
class Order {

    @Id
    @GeneratedValue(generator = "increment")
    @GenericGenerator(name = "increment", strategy = "increment")
    @Column(name = "orderID")
    var orderID: Long = 0

    @Column(name = "status")
    var status: String? = null

    @Column(name = "createdDate")
    var createDate: Date = Date()

    @Column(name = "destination")
    var destination: String? = null

    @Column(name = "userID")
    var userID: Long = 0

    @Column(name = "businessID")
    var businessID: Long = 0

    @Column(name = "goods")
    var goods: String? = null
}